plugins/importexport/pubmed/tests/functional/FunctionalPubmedExportTest.php

```diff
@@ -13,12 +13,15 @@
  * @brief Test PubMed export.
  */
 
-import('lib.pkp.tests.functional.plugins.importexport.FunctionalImportExportBaseTestCase');
+namespace APP\plugins\importexport\pubmed\tests\functional;
 
-class FunctionalPubmedExportTest extends FunctionalImportExportBaseTestCase
+use PKP\tests\PKPTestCase;
```

Comment:
I was going to fix this test (and another very similar located at the `plugins/generic/crossref` submodule), but then I saw a similar test was moved to Cypress, and got in doubt.

Reply:
That's fine -- once the unit test stuff is cleaned up (and this is a big help) we should consider more broadly what do do with PHPUnit and what to do with Cypress.

Reply:
I think Cypress is the most important, it assures the software works as it should.
Unit tests are faster to run, better scoped and can test things that might not be accessible (or perhaps too complex to simulate) through the interface, such as the little details.
It would be great to have both 😁